Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -2244,6 +2248,366 @@ - (void)optionalFormButtonTapped:(id)sender {
     [self beginTaskWithIdentifier:OptionalFormTaskIdentifier];
 }
 
+#pragma mark - Predicate Tests
+/*
+ This is intended to test the predicate functions and APIs
+ */
+- (id<ORKTask>)makePredicateTestsTask {
+    NSMutableArray *steps = [NSMutableArray new];
+    
+    {
+        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"intro_step"];
+        step.title = @"Predicate Tests";
+        [steps addObject:step];
+    }
+    
+    // Test Expected Boolean value
+    {
+        ORKQuestionStep *step = [ORKQuestionStep questionStepWithIdentifier:@"question_01"
+                                                                      title:@"Choose Yes to pass this question, or No to Fail it"
+                                                                     answer:[ORKAnswerFormat booleanAnswerFormat]];
+        step.optional = NO;
```

Comment:
Maybe make all of them optional? That way we have easier skipping if we only are interested in testing something in particular.